awscli/customizations/configure/__init__.py

```diff
@@ -36,3 +36,13 @@ def mask_value(current_value):
         return 'None'
     else:
         return ('*' * 16) + current_value[-4:]
+
+
+def profile_to_section(profile_name):
+    """Converts a profile name to a section header to be used in the config."""
+
+    if ' ' in profile_name:
```

Comment:
It's probably better to use `string.whitespace` to account for tabs an other weird edge cases.

Reply:
I'm okay with making this a little more robust but I'm concerned about what we view as valid whitespace in a section header. A profile name with a newline in it is probably invalid.

Reply:
It's more about being coordinated with what `shlex.split` uses in botocore.